docs/core/linux-prerequisites.md

```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
 
 ### Scripting Installs with the .NET Core installer script
 
-The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the script from: https://dot.net/v1/dotnet-install.sh
+The [`dotnet-install` scripts](./tools/dotnet-install-script.md) are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the script from [https://dot.net/v1/dotnet-install.sh](https://dot.net/v1/dotnet-install.sh).
```

Comment:
might be better to remove the inline code from `dotnet-install`. It looks a bit weird in the middle of a link:
![image](https://user-images.githubusercontent.com/12971179/38643421-e017ca98-3d91-11e8-94bc-200a46ed7e6e.png)